spec/authentication.yml

```diff
@@ -0,0 +1,823 @@
+              - "base64"
+            default: "application/json"
+        requestBody:
+          description: "API Key"
+          required: true
+          content:
+            text/plain:
+              schema:
+                $ref: 'openapi.yml#/components/schemas/ApiKey'
+
+        responses:
+          "200":
+            $ref: 'openapi.yml#/components/responses/AccessTokenGeneric'
+          "400":
+            $ref: 'openapi.yml#/components/responses/BadRequest'
+          "401":
+            $ref: 'openapi.yml#/components/responses/UnauthorizedError'
+          "404":
+            $ref: 'openapi.yml#/components/responses/ResourceNotFound'
+          "500":
+            $ref: 'openapi.yml#/components/responses/InternalServerError'
+
+    AWSAuthenticate:
+      post:
+        tags:
+        - "authentication"
+        summary: "Get a short-lived access token for applications running in AWS."
+        description: "The access token is used to communicate to the REST API that
+          the bearer of the token has been authorized to access the API and perform
+          specific actions specified by the scope that was granted during authorization.
+
+          For API usage, the base64-encoded access token is ordinarily passed as an HTTP
+          Authorization header as `Authorization: Token token=<base64-encoded token>`.
+
+          The `login` must be URL encoded and the host ID must have the prefix
+          `host/`. For example, the host webserver would login as `host/webserver`,
+          and would be encoded as `host%2Fwebserver`.
+
+          The `service_id`, if given, must be URL encoded. For example,
+          `prod/gke` must be encoded as `prod%2Fgke`.
+
+          For detailed instructions on authenticating to Conjur using this endpoint,
+          reference the documentation:
+          [AWS IAM Authenticator](https://docs.conjur.org/Latest/en/Content/Operations/Services/AWS_IAM_Authenticator.htm) (`authn-iam`)."
+        operationId: "getAccessTokenViaAWS"
+        parameters:
+        - name: "service_id"
+          in: "path"
+          required: true
+          description: "URL-encoded authenticator service ID"
+          schema:
+            type: string
+          example: "prod/okta"
+        - name: "account"
+          in: "path"
+          required: true
+          description: "Organization account name"
+          schema:
+            $ref: 'openapi.yml#/components/schemas/AccountName'
+        - name: "login"
+          in: "path"
+          required: true
+          description: "URL-encoded login name. For hosts, the login name is `host/<host-id>`"
+          schema:
+            $ref: '#/components/schemas/LoginName'
+        - name: "Accept-Encoding"
+          in: "header"
+          description: "Setting the Accept-Encoding header to base64 will return a pre-encoded access token"
+          schema:
+            type: "string"
+            enum:
+              - "application/json"
```

Comment:
Does setting this value to `application/json` change the behavior in Conjur?

Reply:
hm, I don't know. it's certainly different than `base64`. I don't think I added this, I think it was there.

Reply:
Okay, I will look into this and remove the extra option if it isn't needed. This also clears up my other question about the enum with just `base64` in it.